src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +41,51 @@
      * Андрей - 600
      * София - 900</p>
      */
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
constant's names should be in uppercase

Reply:
Maven doesn't let me do this in uppercase